tempesta_fw/t/unit/test_frang.c

```diff
@@ -0,0 +1,688 @@
+/**
+ *  		Tempesta FW
+ *
+ * Copyright (C) 2012-2014 NatSys Lab. (info@natsys-lab.com).
+ * Copyright (C) 2015 Tempesta Technologies, Inc.
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License,
+ * or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * his program; if not, write to the Free Software Foundation, Inc., 59
+ * Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+ */
+
+#include "test.h"
+#include "sched_helper.h"
+#include <linux/ctype.h>
+#include <linux/hashtable.h>
+#include <linux/spinlock.h>
+#include <linux/file.h>
+#include <linux/fs.h>
+#include <linux/slab.h>
+#include <linux/inet.h>
+#include <linux/module.h>
+#include <linux/printk.h>
+#include <net/ipv6.h>
+#include <net/sock.h>
+//#include "/usr/include/arpa/inet.h"
+#include <linux/socket.h>
+//#include <arpa/inet.h>
+#include <linux/slab.h>
+//#include <netinet/in.h>
+#include "../../tempesta_fw.h"
+#include "../../lib.h"
+#include "../../log.h"
+#include "../../msg.h"
+#include "../../http.h"
+#include "cfg.h"
+#include "../../gfsm.h"
+#include "../../client.h"
+#include "../../connection.h"
+//#ifdef module_init
+//#undef module_init
+//#endif
+
+//#ifndef _INCLUDE_FRANG_CODE
+//#define _INCLUDE_FRANG_CODE
+//#include <linux/module.h> 
+//#include "../../classifier/frang.c"
+//#endif
+//#undef module_init
```

Comment:
Our coding style prohibits C++-like comments.
We don't leave dead code as commented out, but rather just delete it.